.circleci/config.yml

```diff
@@ -240,120 +240,91 @@ jobs:
-  # Deploy task.
-  deploy:
+
+  # Development build: all dof-dss packages use HEAD on development branch, pushes to fixed non-integrating branch.
+  dev_build:
     docker:
-      - image: circleci/php:7.3.14-apache-node-browsers
+      - image: circleci/php:7.3.14-apache-browsers
+    environment:
+      # git variables to avoid empty committer identity errors
+      EMAIL: "circleci@localhost"
+      GIT_COMMITTER_NAME: "Circle CI"
+      GIT_AUTHOR_NAME: "Circle CI"
     steps:
-      - checkout_code
+      - attach_workspace:
+          at: ./
+      - install_php_os_extensions
       - run:
-          name: Add platform.sh hosts to SSH known hosts
-          command: mkdir -p ~/.ssh && ssh-keyscan -H git.uk-1.platform.sh >> ~/.ssh/known_hosts
+          name: Add extra OS and PHP extensions/config
+          command: |
+            sudo docker-php-ext-install pcntl posix
       - run:
-          name: Relay to platform.sh
+          name: Keyscan Platform.sh region hostnames + GitHub for password-less access
           command: |
-            # Code is checked out under /home/circleci/project
-            cd /home/circleci/project
-            git remote add platform ${PLATFORM_REPO_URL}
-            git push -f platform ${CIRCLE_BRANCH}
+            mkdir -p ~/.ssh
+            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
+            ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+      - run:
+          name: Install the Platform.sh CLI tool
+          command: |
+            curl -sS https://platform.sh/cli/installer | php
+      - run:
+          name: Re-point dof-dss packages to use latest development code and push.
+          command: |
+            cd ~/project
+            git checkout -b $DEV_BUILD_BRANCH
 
-  # Our most expensive, full stack tests. Requires up to date content and services for chromedriver, yarn and database.
-  ui_tests:
-    docker:
-      - image: circleci/php:7.3.14-apache-node-browsers
-      - image: circleci/mysql:5.7.27
-      - image: drupalci/chromedriver:production
-        environment:
-          CHROMEDRIVER_WHITELISTED_IPS: ""
-          CHROMEDRIVER_URL_BASE: "/wd/hub"
-    steps:
-      - attach_workspace:
-          at: ./
+            composer require dof-dss/nidirect-site-modules:dev-development dof-dss/nidirect-migrations:dev-development \
+              dof-dss/nidirect-d8-test-install-profile:dev-development dof-dss/nicsdru_origins_theme:dev-development \
+              dof-dss/nicsdru_origins_modules:dev-development dof-dss/nicsdru_nidirect_theme:dev-development
+
+            git add composer.*
+            git commit -m "Set dof-dss packages to HEAD development for build"
+
+            # Push to fixed, non-integrating build branch. GitHub webhook integration will propagate this
+            # to platform.sh for later steps to use.
+            git push -f origin $DEV_BUILD_BRANCH
+
+            # Pause for webhook to propagate the change from GH to PSH.
```

Comment:
Might not be needed, was there originally to act as a pause before trying to sync data from master. Given that any CI checks can take 10 mins and the deploy task takes around 2 minutes, this is why the sync task has been decoupled to its own workflow, timed to run at a later point.